app/src/main/java/com/example/capstone/ProfileCreationFragment.java

```diff
@@ -22,12 +65,119 @@ public static ProfileCreationFragment newInstance(String param1, String param2)
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        mAuth = FirebaseAuth.getInstance();
     }
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
         return inflater.inflate(R.layout.fragment_profile_creation, container, false);
     }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        etFullName = getActivity().findViewById(R.id.et_full_name);
+        etSchool = getActivity().findViewById(R.id.et_school);
+
+        ivProfilePicture = getActivity().findViewById(R.id.iv_profile_picture);
+        ivProfilePicture.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.d(TAG, "Clicked");
+                onPickPhoto(v);
+            }
+        });
+
+        Glide.with(getActivity()).load(R.drawable.placeholder_image).into(ivProfilePicture);
+
+        btnCompleteProfile = getActivity().findViewById(R.id.btn_complete_profile);
```

Comment:
Line 82 and 93 should be together right after line 80. So we're grouping all `findViewById` code together.